signer/src/transaction_coordinator.rs

```diff
@@ -221,8 +226,31 @@ where
             .await
     }
 
+    async fn is_epoch3(&mut self) -> Result<bool, Error> {
+        if self.is_epoch3 {
+            return Ok(true);
+        }
+        tracing::info!("Checked for whether we are in Epoch 3 or later");
```

Comment:
these should prob be debug
but otherwise nice function :+1: much cleaner than what was there

Reply:
done in https://github.com/stacks-network/sbtc/pull/772/commits/31f87cab56525a908c925becaf8f709abf9e4b60.